test/benzene/valgrind-integral/test

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+import os
+import sys
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..'))
+
+from runtest import version_info, get_filter, cli, run
+from runtest_config import configure
+
+assert version_info.major == 2
+
+os.environ["GIMIC_LAUNCHER"] = "valgrind --tool=memcheck --log-file=valgrid-summary.dat --leak-check=full --track-origins=yes --show-reachable=yes -v"
+
+options = cli()
+
+f = [
+     get_filter(from_string='in use at exit',
+                num_lines=1,
+                rel_tolerance=0),
+     get_filter(from_string='ERROR SUMMARY',
+               to_string='errors from',
+               rel_tolerance=0),
+     ]
+acc = [
+     get_filter(from_string=' *** Integrating current',
+               num_lines=10,
+               rel_tolerance=1.0e-8),
+    ]
+
+ierr = run(options,
+           configure,
+           input_files=['gimic.inp'],
+           filters={'valgrid-summary.dat': f,
+	            'stdout': acc})
```

Comment:
I would probably not check the Valgrind output through the entire runtest infrastructure but here write a short custom Python function which only checks that there are no errors and not leaks.

Reply:
I'm not good at Python, so maybe I'm missing something. The output from Valgrind needs to be parsed in some way or another. If I had to do it in a separate script, then wouldn't it have to replicate what runtest does?

Reply:
OK - I don't want to complicate it. Finish what you had in mind, later we can always adjust if really needed but it's probably better as you have done.